rust-manifest/tests/channel-rust-nightly-example.toml

```diff
@@ -0,0 +1,52 @@
+manifest-version = "2"
+date = "2015-10-10"
+[pkg.rust]
+  version = "rustc 1.3.0 (9a92aaf19 2015-09-15)"
+  [pkg.rust.target.x86_64-unknown-linux-gnu]
+    available = true
+    url = "example.com"
+    hash = "..."
+    [[pkg.rust.target.x86_64-unknown-linux-gnu.components]]
+      pkg = "rustc"
+      target = "x86_64-unknown-linux-gnu"
+    [[pkg.rust.target.x86_64-unknown-linux-gnu.components]]
+      pkg = "rust-docs"
+      target = "x86_64-unknown-linux-gnu"
+    [[pkg.rust.target.x86_64-unknown-linux-gnu.components]]
+      pkg = "cargo"
+      target = "x86_64-unknown-linux-gnu"
+    [[pkg.rust.target.x86_64-unknown-linux-gnu.components]]
+      pkg = "rust-std"
+      target = "x86_64-unknown-linux-gnu"
+    # extensions are rust-std or rust-docs that aren't in the rust tarball's component list
+    [[pkg.rust.target.x86_64-unknown-linux-gnu.extensions]]
+      pkg = "rust-std"
+      target = "x86_64-unknown-linux-musl"
+    [[pkg.rust.target.x86_64-unknown-linux-gnu.extensions]]
+      pkg = "rust-std"
+      target = "i686-unknown-linux-gnu"
+[pkg.rustc]
+  version = "rustc 1.3.0 (9a92aaf19 2015-09-15)"
+  [pkg.rustc.target.x86_64-unknown-linux-gnu]
+    available = true
+    url = "example.com"
+    hash = "..."
+[pkg.cargo]
+  version = "cargo 0.4.0-nightly (553b363 2015-08-03)"
+  [pkg.cargo.target.x86_64-unknown-linux-gnu]
+    available = true
+    url = "example.com"
+    hash = "..."
+[pkg.rust-std]
+  version = "rustc 1.3.0 (9a92aaf19 2015-09-15)"
+  [pkg.rust-std.target.x86_64-unknown-linux-gnu]
+    available = true
+    url = "example.com"
+    hash = "..."
+[pkg.rust-docs]
+  version = "rustc 1.3.0 (9a92aaf19 2015-09-15)"
+  [pkg.rust-docs.x86_64-unknown-linux-gnu]
```

Comment:
Shouldn't this be `rust-docs.target.x86_64-unknown-linux-gnu`? If so, shouldn't it cause the tests to fail?